src/componentes/mounted/banner/BannerWebinar.tsx

```
@@ -1,24 +1,26 @@
         setDisplayBannerWebinar(!!localStorage.getItem("DisplayBannerWebinar"))
-    },[display])
+    },[display]);
+    
     return !DisplayBannerWebinar && <ModalAlertDisplay
         displayStates = {{display, setDisplay}}
         Child = {Alert_v4}
         childProps = {{
             titulos : {
-                Titulo : "Transição de gestão da Saúde Municipal",
-                SubTitulo : "O papel dos profissionais e gestores da APS:"
+                Titulo : "Gestão em Saúde",
```

Biome (1.9.4)

[error] This object property name should be in camelCase.



(lint/style/useNamingConvention)